controller/api/models.py

```diff
@@ -381,7 +381,7 @@ def _healthcheck(self, containers, config):
         # HACK (bacongobbler): we need to wait until publisher has a chance to publish each
         # service to etcd, which can take up to 20 seconds.
         time.sleep(20)
-        for i in range(0, 4):
+        for i in range(len(intervals)):
```

Comment:
`xrange` probably doesn't matter for such a short list, but I use it consistently anyway, so :+1: